src/utils/email.tsx

```diff
@@ -1,13 +1,31 @@
 import Email from "@/components/email/Email";
 import { Resend } from "resend";
 
+interface SendEmailParams {
+  email: string;
+  id: string;
+  name: string;
+  position: string;
+  subject: string;
+  preview: string;
+}
+
 const resend = new Resend(process.env.RESEND_API_KEY);
 
-const send = async ({ email, id, name, position, subject, preview }) => {
+const send = async ({
+  email,
+  id,
+  name,
+  position,
+  subject,
+  preview,
+}: SendEmailParams) => {
   const { data, error } = await resend.emails.send({
     from: "Hackathon <info@hackathon.com>",
     to: [email],
     subject: subject,
+    text: `Dear ${name},\n\ Here is a preview to where to meet \n\ Happy Hacking
```

Comment:
no new information should be present here, this pr should only be focusing on the types